visualize.go

```diff
@@ -62,27 +63,28 @@ func (o visualizeErrorOption) applyVisualizeOption(opt *visualizeOptions) {
 }
 
 func updateGraph(dg *dot.Graph, err error) error {
-	var errors []errVisualizer
+	var errs []errVisualizer
 	// Unwrap error to find the root cause.
 	for {
 		if ev, ok := err.(errVisualizer); ok {
-			errors = append(errors, ev)
+			errs = append(errs, ev)
 		}
-		e, ok := err.(causer)
-		if !ok {
+		e := errors.Unwrap(err)
+		if e == nil {
 			break
+		} else {
+			err = e
```

Comment:
with `break`, you don't need this else block.

https://github.com/uber-go/guide/blob/master/style.md#reduce-nesting